docs/en-US/component/select-v2.md

```diff
@@ -153,7 +153,6 @@ select-v2/remote-search
 | persistent                        | when select dropdown is inactive and `persistent` is `false`, select dropdown will be destroyed                                          | boolean                            | true / false        | true          |
 | popper-options                    | Customized popper option see more at [popper.js](https://popper.js.org/documentation.html)                                               | object                             | -                   | -             |
 | automatic-dropdown                | for non-filterable Select, this prop decides if the option menu pops up when the input is focused                                        | boolean                            | -                   | false         |
-| clear-icon                        | Customized clear icon component                                                                                                          | string / Component                 | —                   | CircleClose   |
```

Comment:
Could you please help correct the lower-cased `component` to be `Component`?

https://github.com/element-plus/element-plus/blob/4902a7ac3c97d101c9aeaa6eb55ce740bf3e2a16/docs/en-US/component/select-v2.md?plain=1#L138